lib/main/atom/tooltipManager.ts

```diff
@@ -15,7 +15,10 @@ import escape = require("escape-html")
 
 export function getFromShadowDom(element: JQuery, selector: string): JQuery {
   var el = element[0]
-  var found = (<any>el).rootElement.querySelectorAll(selector)
+  var found = (<any> el).querySelectorAll(selector)
+  if (!found) {
+    found = (<any> el).rootElement.querySelectorAll(selector)
+  }
```

Comment:
Regarding 3, does it blow up though? Haven't checked.

I have no clue on the typeset you are using, what would be the best approach to get rid of the `<any>`?

I'll do N°1